cmd/clusterctl/hack/local-overrides.py

```diff
@@ -140,7 +141,7 @@ def create_local_overrides():
         assert p is not None, 'invalid configuration: please specify the configuration for the {} provider'.format(provider)
 
         repo = p.get('repo', '.')
-        config_folder = p.get('configFolder', 'config')
+        config_folder = p.get('configFolder', 'config/default')
```

Comment:
After the changes introduced by https://github.com/kubernetes-sigs/cluster-api/pull/2279 the root folder for generating manifest is `config` (not anymore `config/default`), so this value should not be changed.

Reply:
Thank you @fabriziopandini I will try to fix this in CAPA then.